test-resources/e2e-test-volumes/test-uuid/cronjob.test.id

```diff
@@ -0,0 +1 @@
+7c30ff2f-b2a3-4125-b06d-a8cb21e97c15
```

Comment:
This file should not be here. Please remove from the PR and from your local file system.

How this probably happened: Before 051f84be8856b5fe5c352f364939b589f63e888d, these files were generated by e2e tests and placed on disk, and there was also a matching .gitignore. This mechanism was removed together with the .gitignore file. You probably ran the tests before that commit and the file was still around on your disk when you did `git add .`.

Reply:
Amended in 4a36820